### ISSUE TITLE
1st-party tracker blocking with Chrome

### ISSUE THREAD

**Issue by mikhoul** [NanoAdblocker/NanoCore#296]
Hi @jspenguin2017 ,

Are you going to find a solution for the 1st-party tracker in Chrome with Nano ?  

https://github.com/uBlockOrigin/uBlock-issues/issues/780

Regards  :octocat:

**Comment by llacb47** [comment]
When it comes to uBo stable it will come to Nano, most likely.

**Comment by xlollomanx** [comment]
@llacb47 Yes, but if I'm not wrong that feature will work 100% only on firefox. If someone try to implement it on chrome it will probably broke their policy. And in addition chrome will probably ditch all adblockers soon because of manifestv3. This cahnge is still in developing so let's see how it will end but likely you have to leave chrome soon. I'm using edge chromium atm and I hope microsoft will still support adblockers, if not I'll found another browser.

**Comment by jspenguin2017** [comment]
I'm not very familiar with the implementation details of the feature upstream, but from what I've seen so far, I don't really like how it's implemented. I think existing filters are not written to expect resolved (or uncloaked) CNAMEs, and a user-maintained `cnameAliasList` in the advanced settings page is not going to scale very well. 

As for that specific issue, it can probably be fixed with whitelist-only rules (block all subdomains except). It needs to be handled on a case by case basis but I think there might be ways to automate it.

**Comment by jspenguin2017** [comment]
@llacb47 
The code will be synchronized over, but the feature doesn't work on Chromium due to missing extension APIs.

**Comment by jspenguin2017** [comment]
@xlollomanx 
Be sure to cite your sources, "read somewhere" is not convincing enough as an argument. Also, don't go off topic, discussion about Manifest V3 is [here](https://github.com/NanoAdblocker/NanoCore/issues/238).
